styles/landing.css

```diff
@@ -30,11 +30,20 @@ body.landing {
 	-moz-box-sizing: border-box;
 	box-sizing: border-box; 
 }
- 
 .point {
 	position: relative;
 	padding: 2rem;
 	text-align: center;
+	opacity: 0;
+	-webkit-transform: scaleX(0.1) translateY(0);
+	-moz-transform: scaleX(0.1) translateY(0);
+	transform: scaleX(0.1) translateY(0);
+	-webkit-transition: all 1.5s ease-in-out;
+	-moz-transition: all 1.5s ease-in-out;
+	transition: all 1.5s ease-in-out;
+	-webkit-transition-delay: 1s;
+	-moz-transition-delay: 1s;
+	transition-delay: 1s;
```

Comment:
this is actually not a direct copy of the instructions; i adjusted the pace of the animation a bit so i could actually see it.